spec/comm.mediawiki

```diff
@@ -189,13 +192,19 @@ of their penalization.
 {|
 ! field   !! type   !! description
 |-
-| matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
+| activematches || <nowiki>[object]</nowiki> || list of active [[orders.mediawiki/#Match_negotiation|Match objects]]
 |-
-| order statuses || <nowiki>[object]</nowiki> || list of <code>Order Status Object</code>
+| activeorderstatuses || <nowiki>[object]</nowiki> || list of active <code>Order Status Object</code>
 |-
 | score || int || The user's score. The math behind this is currently being adjusted
 |-
-| suspended || bool || true if suspended. will be deprecated in a future version
+| tier || int || Bond tier. One or more indicates the account can trade. Not set for legacy accounts.
```

Comment:
This would be "Not set by legacy servers." because it is set even for legacy accounts if the server has updated.

A legacy account (that paid the old reg fee) gets a free and perpetual tier +1, as you've noted below on `legacyFeePaid`.

Reply:
Updated the wording.